DokanNet.Tests/DokanNet.Tests.csproj

```diff
@@ -15,7 +15,7 @@
     net4.6              net4.6
     netstandard1.3      <missing>
     -->
-    <TargetFrameworks>net45;net46</TargetFrameworks>
+    <TargetFrameworks>net45;net46;netcoreapp3.1</TargetFrameworks>

```

Comment:
It looks like the build does not pass on the CI 😢 
https://ci.appveyor.com/project/Liryna/dokan-dotnet/builds/42074445/job/239r9qpoimordkf9

Reply:
There is something strange going on with the test project. I can run it with "dotnet test" from command line but from within Visual Studio I always get the error message "Unable to read project information for 'DokanNet.Tests': Sequence contains more than one element" and when trying to rebuild it causes error messages about target frameworks not in project.assets.json, but that is most likely just a cause of the original issue. I would think that there is something wrong somewhere in the project files etc but I cannot figure out exactly what that could be.

Reply:
I have the same error now even if I remember this was working previously.
@kyanha has even reported the issue to Microsoft here https://developercommunity.visualstudio.com/t/error-occurred-while-restoring-nuget-packages-the-3/761782
I am still using Version 16.11.5, on which version are you? maybe the latest one has a fix ?
Otherwise....I was just locally setting one framework and it worked

Reply:
I am using Visual Studio 2022 17.0.4. I get the same errors when I try in 2019 16.11.8.

For now we could of course set just one framework if that helps. In the future it would be great to test more than one framework though. (I have for example ideas about using IDokanOperationsUnsafe with managed Stream objects using Read and Write overloads for Span<byte> which only works in .NET Core and .NET Standard 2.1 and not in .NET Framework.)

Reply:
Thanks for doing the test. Looks like there is not much we can do besides adding a comment to retry this later so :(

Reply:
Thanks for the change! FYI the release CI now fails due to a memory corruption https://ci.appveyor.com/project/Liryna/dokan-dotnet/builds/42080843/job/lnx5iyhcndi7kdni

Reply:
That is interesting and I saw that now too when I retried "dotnet test" on command line. I'll try to figure out what is happening here.